src/newViews/ProjectEdit/GeoAreas/RegionCard/AddAdminLevel/index.tsx

```diff
@@ -0,0 +1,171 @@
+import React, { useMemo, useCallback, useState } from 'react';
+import {
+    _cs,
+} from '@togglecorp/fujs';
+import { IoAdd } from 'react-icons/io5';
+import {
+    ContainerCard,
+    Button,
+    ConfirmButton,
+    Tabs,
+    TabList,
+    TabPanel,
+    Tab,
+    List,
+} from '@the-deep/deep-ui';
+
+import { useRequest, useLazyRequest } from '#utils/request';
+import notify from '#notify';
+import {
+    AdminLevelGeoArea,
+    MultiResponse,
+} from '#typings';
+
+import AddAdminLevelForm from './AddAdminLevelForm';
+import styles from './styles.scss';
+
+interface TabDetails {
+    name: string;
+    children: string;
+    data?: AdminLevelGeoArea,
+}
+const defaultTab: TabDetails[] = [
+    {
+        name: 'adminLevel1',
+        children: 'Admin Level 1',
+    },
+];
+
+const tabKeySelector = (d: TabDetails) => d.name;
+
+interface Props {
+    activeRegion: number;
+}
+
+function AddAdminLevel(props: Props) {
+    const {
+        activeRegion,
+    } = props;
+
+    const [activeTab, setActiveTab] = useState('adminLevel1');
+    const [tab, setTab] = useState<TabDetails[]>(defaultTab);
```

Comment:
Let's have this empty by default and set the defaultTab if response.count is zero on /admin-levels/ query.